modules/programs/git.nix

```diff
@@ -340,6 +340,15 @@ in {
           '';
         };
 
+        pagerOpts = mkOption {
+          type = types.listOf types.str;
+          default = [ "--tabs=4" "-RFX" ];
+          apply = concatMapStringsSep " " escapeShellArg;
```

Comment:
```suggestion
          apply = escapeShellArgs;
```

Reply:
Well, actually I'd rather put this at the use site rather than in `apply`, so that pager options can still be inspected as a list.

Reply:
Done.